tox.ini

```diff
@@ -50,7 +50,7 @@ commands =
     rst2html.py --report=info --halt=warning README.rst /dev/null
     rst2html.py --report=info --halt=warning CHANGES.rst /dev/null
 
-[py311-latest-noi18n]
+[py311-django50-latest-noi18n]
```

Comment:
I think this section can be removed and just use a  conditional setenv in the standard section:
```
    noi18n: DJANGO_SETTINGS_MODULE = django_countries.tests.settings_noi18n
```